composer.json

```diff
@@ -2,18 +2,18 @@
   "name": "magento/composer",
   "description": "Magento composer library helps to instantiate Composer application and run composer commands.",
   "type": "library",
-  "version": "1.2.0",
+  "version": "1.3.0",
   "license": [
     "OSL-3.0",
     "AFL-3.0"
   ],
   "require": {
-    "php": "~5.5.0|~5.6.0|~7.0.0|~7.1.0",
-    "composer/composer": "1.4.1",
-    "symfony/console": "~2.3, !=2.7.0"
+    "php": "~7.1.3|~7.2.0",
+    "composer/composer": "~1.6.0",
```

Comment:
Since this PR was accepted: https://github.com/magento/magento2/pull/14013
Shouldn't the composer constraint be changed to `^1.6` instead of `~1.6.0`?
Otherwise composer won't upgrade to version 1.7 or 1.8 or higher when they are released, which was the purpose of the above mentioned PR.

/cc: @sandermangel, @miguelbalparda